arch/arm/core/fault.c

```diff
@@ -37,6 +37,69 @@
 	} while ((0))
 #endif
 
+#if defined(CONFIG_ARM_SECURE_FIRMWARE)
+
+/* Exception Return (EXC_RETURN) is provided in LR upon exception entry.
+ * It is used to perform an exception return and to detect possible state
+ * transition upon exception.
+ */
+
+/* Prefix. Indicates that this is an EXC_RETURN value.
+ * This field reads as 0b11111111.
+ */
+#define EXC_RETURN_INDICATOR_PREFIX     (0xFF << 24)
```

Comment:
Doesn't CMSIS contain anything about the `EXC_RETURN` mappings?

Reply:
No, it does not.

Reply:
Do you mind filing an issue on the CMSIS_5 project

Reply:
https://github.com/ARM-software/CMSIS_5/issues/340

for both EXC_RETURN and the Integrity Signature!